js/script.js

```diff
@@ -0,0 +1,14 @@
+const navlinks = document.querySelector('.navlinks');
+const menu = document.querySelector('.fa-bars');
+const close = document.querySelector('.fa-x');
+// console.log(menu)
+
+menu.addEventListener('click', () => {
+  navlinks.classList.add('open');
+  close.classList.add('show');
+});
+
```

Comment:
- [ ] Great job with the opening and closing functionality. There are some issues remaining to match the project requirements. As stated in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md), we are supposed to close the menu when clicking on one of the links. Kindly implement it accordingly. 🤝️

**Project requirements**
![image](https://user-images.githubusercontent.com/17243325/218032986-2dbc422d-7cc1-48ad-a25b-6e8af961fd61.png)

- [ ] One of the objectives of this project is to understand how to parse a Figma design precisely. Let's stick to the design as much as possible. Please update the background color and position of the links. The close button is looking a bit bigger on your design, please check the  `font-size` too. 😎️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/218033498-3874973b-f7e4-4c30-9233-c099c1cc44ab.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/218033615-eb1ff811-6a45-41e6-af96-c7b56910f370.png" width="300">|